developers/Holmakefile

```diff
@@ -1,7 +1,30 @@
+INCLUDES = ../compiler/bootstrap/compilation/x64/64
```

Comment:
I worry that this is a circular dependency. Or at least an undesirable one, since `developers` may be required for some very basic stuff...

(The way around this would be to move these dependencies to a separate directory.)